core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -398,6 +403,42 @@ public void getBlockInfo() throws Exception {
     assertEquals(expectedBlockInfo, mBlockMaster.getBlockInfo(blockId));
   }
 
+  @Test
+  public void getNewContainerId() throws Exception {
+    int total = 10000;
```

Comment:
Could you also make the variable name more descriptive please?
```suggestion
    final int total = 10_000;
```